communication/slack-guidelines.md

```diff
@@ -1,90 +1,213 @@
+a great time to enlist the help of another admin.
+
+Try to take any situations that involve upset or angry members to DM or video
+chat. Please document these interactions for other Slack admins to review.
+
+Content will be automatically removed if it violates code of conduct or is a
+sales pitch. Admins will take a screenshot of such behavior in order to document
+the situation. The community takes such violations extremely seriously, and the
+admins are empowered to handle it swiftly.
+
+
+### Sending Messages to the Channel
+
+`@here` and `@channel` should be used rarely. Members will receive notifications
+from these commands. Remember Kubernetes is a global project - please be kind.
+Note: `@all` is restricted and can only be used by admins.
+
+
+### Reviewing Slack Requests
+
+Admins are tasked with reviewing requests for channels and other things such as
+bots, tokens or webhook requests.
+
+
+#### Reviewing Channel Requests
+
+Channel requests should be reviewed for their relation and relevance to the
+Kubernetes community. Typically channels should be dedicated to SIGs, WGs, UGs,
+sub-projects, community topics, and other things related to Kubernetes programs
+and projects.
+
+Small external projects are encouraged to use the channel of the SIG, WG, or UG
+most relevant to them. Other things such as programming language specific
+channels are discouraged and should in turn be steered to `#kubernetes-client`
+or communication avenues commonly used by their specific language.
+
+In general, use your best judgment.
+
+
+#### Reviewing Bot, Token, or Webhook requests
+
+Requests should first be evaluated for their relevance to the project. Typically
+approved requests are related to: GitHub, the CNCF, or other tools/platforms
+used to aid in the management of Slack. Requests outside of this scope should be
+heavily scrutinized and reviewed for **ANY** potential security, privacy, or
+usability concerns.
+
+It is best to err on the side of not allowing a Bot, Token or Webhook request
+than allowing one.
+
+
+### Inactivating Accounts
+
+For the reasons listed below, admins may inactivate individual Slack accounts.
+Due to Slack’s framework, it does not allow for an account to be banned or
+suspended in the traditional sense, merely inactivated. See [Slack’s policy on
+inactivated accounts] for more information.
+
+- Spreading spam content in DMs and/or channels.
+- Not adhering to the code of conduct set forth in DMs and/or channels.
+- Overtly selling products, related or unrelated to Kubernetes.
+
+---
+
+## Requesting a Channel
+
+- Create a [GitHub Issue] using the Slack Request template.
+- Include the desired name of the channel. Typical naming conventions follow:
+  - `#kubernetes-foo`
+  - `#sig-foo`
+  - `#meetup-foo`
+  - `#location-user`
+  - `#projectname`
+- In the issue describe the purpose of the Channel. Channels should be
+  dedicated to [SIGs, WGs, UGs], sub-projects, community topics, or related
```

Comment:
admins can validate the request by visiting sigs-list.md or some other community resource like k-dev ML if it makes sense to have a 'where to find this info' for new admins

Reply:
Added it to the request processing section.